lib/foreman_bootdisk/engine.rb

```diff
@@ -57,6 +57,11 @@ class Engine < ::Rails::Engine
         provision_method 'bootdisk', N_('Boot disk based')
         template_labels 'Bootdisk' => N_('Boot disk embedded template')
         allowed_template_helpers :bootdisk_chain_url, :bootdisk_raise
+
+        extend_page "subnets/index" do |cx|
+          cx.add_pagelet :modals, partial: 'subnets/bootdisk_modal'
+          cx.add_pagelet :title_action_buttons, name: 'Bootdisk', partial: 'subnets/index_bootdisk_button', priority: 10000
+        end
       end
     end
 
```

Comment:
These three files do not seem to be related, was this added by mistake?

Reply:
Yes, the mistake - that's happen when you're working on the bootdisk improvements at many fronts :/